### PR TITLE
feat(tier4_planning_launch): create parameter yaml for behavior_velocity_planner

### PR DESCRIPTION
**Pull request by h-ohta**
## Description

- create parameter yaml for behavior_velocity_planner

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_planning_launch/config/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/behavior_velocity_planner.param.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/**:
+  ros__parameters:
+    launch_stop_line: false
```
Comment (taikitanaka3) [inline launch/tier4_planning_launch/config/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/behavior_velocity_planner.param.yaml:3]: [Q] mistake?
```suggestion
    launch_stop_line: true
```

**Comment by taikitanaka3** [comment]
@h-ohta 
can you add these `max_jerk`=2.5 and `system_delay`=0.5 argument by default too?

https://github.com/autowarefoundation/autoware.universe/blob/93d95dd63a9d84bdca4fd87acb4b200098dd8bf3/planning/behavior_velocity_planner/include/behavior_velocity_planner/planner_data.hpp#L64

**Comment by h-ohta** [comment]
@taikitanaka3 I fixed.

**Review by taikitanaka3 — APPROVED** [review]
LGTM

**Pull request merged**